packages/venia-concept/src/ServiceWorker/Utilities/htmlHandler.js

```diff
@@ -1,3 +1,7 @@
+import { HTML_UPDATE_AVAILABLE } from '@magento/venia-ui/lib/constants/messageTypes';
```

Comment:
Similar comment to a previous one - I don't think `venia-concept` is supposed to pull values from `venia-ui` like this.

Reply:
Yeah need to check with @zetlen about this.